deploy/helm/sumologic/values.yaml

```diff
@@ -429,6 +458,52 @@ fluentd:
     #     mountPath: /certs
     #     readOnly: true
   
+    ## Output configuration
+    ## tag: tag for fluentd match
+    ## id: sumologic output @id
+    ## endpoint: [optional] [output key by default] It should match the sumologic.sources[].endpoint
+    ## drop: [optional] [false by default] Change it to true to drop traffic for specific output
+    ## weight: [optional [0 by default] Order of fluentd output (lower means higher priority)
+    ## The weight has meaning in case of fluentd matches
```

Comment:
ok, I will address it in documentation rather than comment in the values.yaml, but here we go:

Let's say we have two two sources (`source1`, `source2`)

```
      source1:
        tag: prometheus.metrics.custom**
        weight: 90
      source2:
        tag: **
        weight: 80
```
is going to generate in fluentd 
```
# weight 80
<match **>
  # takes all records
</match>
# weight 90
<match prometheus.metrics.custom**>
  # it won't work
</match>
```

but if we switch the weights:

```
      source1:
        tag: **
        weight: 80
      source2:
        tag: prometheus.metrics**
        weight: 90
```
then flunetd config will change to
```
# weight 80
<match prometheus.metrics.custom**>
  # takes custom records
</match>
# weight 90
<match **>
  # takes all other records
</match>
```

Reply:
I'm going to address it in https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/680#issuecomment-634607171

Reply:
```# weight 80
<match **>
  # takes all records
</match>
# weight 90
<match prometheus.metrics.custom**>
  # it won't work
</match>```

Even if the user specified this, the second match won't work because everything was already matched by the catch all block, right?